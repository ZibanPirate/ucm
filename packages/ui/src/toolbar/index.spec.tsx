import { render } from "@testing-library/react";

import { Toolbar } from ".";

describe(`Testing component '${Toolbar.name}' :`, () => {
  it(`should render with 4 div elements`, () => {
    const { container } = render(
      <Toolbar>
        {[1, 2, 3, 4].map((index) => (
          <div key={index}>{index}</div>
        ))}
      </Toolbar>,
    );
    expect(container).toMatchSnapshot();
  });

  it(`should render with 4 div elements and provided 'itemsAlignment' and 'margin'`, () => {
    const { container } = render(
      <Toolbar itemsAlignment="center" margin="1rem">
        {[1, 2, 3, 4].map((index) => (
          <div key={index}>{index}</div>
        ))}
      </Toolbar>,
    );
    expect(container).toMatchSnapshot();
  });
});
