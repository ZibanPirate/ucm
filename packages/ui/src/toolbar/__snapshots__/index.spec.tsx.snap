// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing component 'Toolbar' : should render with 4 div elements 1`] = `
<div>
  <div
    style="display: flex; flex-direction: row; flex: 1; margin: initial; justify-content: start;"
  >
    <div>
      1
    </div>
    <div>
      2
    </div>
    <div>
      3
    </div>
    <div>
      4
    </div>
  </div>
</div>
`;

exports[`Testing component 'Toolbar' : should render with 4 div elements and provided 'itemsAlignment' and 'margin' 1`] = `
<div>
  <div
    style="display: flex; flex-direction: row; flex: 1; margin: 1rem; justify-content: center;"
  >
    <div>
      1
    </div>
    <div>
      2
    </div>
    <div>
      3
    </div>
    <div>
      4
    </div>
  </div>
</div>
`;
